package.json

```diff
@@ -2,6 +2,7 @@
   "name": "template",
   "version": "0.0.0",
   "scripts": {
+    "dev": "concurrently \"npm run dev-client\" \"npm run dev-server\" \"npm run storybook\"",
```

Comment:
This change is unrelated to `Filter page`. I would create a new pull request with this change only.